db/migrations/sqls/20200603101427-add-column-grant-application-client-generated-id-up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE grant_application ADD COLUMN client_generated_id VARCHAR NOT NULL;
```

Comment:
Should this also have a unique constraint on it?

Reply:
Yes :)

Reply:
Fixed in 46d51789fed917cd4d18ed12f2c84b0aac916a15